mteb/evaluation/evaluators/ClassificationEvaluator.py

```diff
@@ -150,10 +162,11 @@ def __init__(
         y_train,
         sentences_test,
         y_test,
-        k=1,
-        batch_size=32,
-        limit=None,
-        **kwargs,
+        task_name: str,
```

Comment:
Do we need to make `task_name` mandatory everywhere? Can we not have it default to `None`? Some people are importing these evaluators from MTEB & we will break their code.

Reply:
I have change it to default to None

Reply:
I have change it to default to None